docs/usage/types.md

```diff
@@ -584,6 +585,28 @@ With proper ordering in an annotated `Union`, you can use this to parse types of
 : type method for constraining strs;
   see [Constrained Types](#constrained-types)
 
+### Exotic Types
+Pydantic also supports exotic types, which are described in detail below:
+
+#### ImportString
+On model instantiation, pointers will be evaluated and imported,
+(distantly related behavior to [PyObject](https://docs.python.org/3/c-api/structures.html#c.PyObject)).
+There is some nuance to this behavior, demonstrated in the examples below.
+
+**Good behavior:**
+{!.tmp_examples/types_import_string_1.md!}
+(This script is complete, it should run "as is")
```

Comment:
Not sure do we need this line here or not. seems the `exec_examples` will add this line:

https://github.com/pydantic/pydantic/blob/dcf35678764be812048fabd4b8f4c577f5c6e267/docs/build/exec_examples.py#L391